docs/web_view_tracking.md

```diff
@@ -0,0 +1,72 @@
+---
+beta: true
+dependencies: 
+- https://github.com/DataDog/dd-sdk-android/blob/alai97/web-view-tracking-android-doc-update/docs/web_view_tracking.md
+title: Android Web View Tracking
+kind: documentation
+description: Monitor web views in your hybrid Android applications.
+further_reading:
+- link: "/real_user_monitoring/android/"
+  tag: "Documentation"
+  text: "Android Monitoring"
+- link: "/real_user_monitoring/browser/"
+  tag: "Documentation"
+  text: "Browser Monitoring"
+---
+
+## Overview
+
+Real User Monitoring allows you to monitor web views and eliminate blind spots in your hybrid Android and Android TV applications.
+
+You can:
+
+- Track user journeys across web and native components in mobile applications
+- Scope the root cause of latency to web pages or native components in mobile applications
+- Support users that have difficulty loading web pages on mobile devices
+
+## Setup
+
+### Prerequisites
+
+Set up the web page you want rendered on your mobile Android and Android TV application with the Browser SDK first. For more information, see [RUM Browser Monitoring][1].
+
+### Update your existing SDK setup
+
+1. Download the [latest version][2] of the RUM Android SDK.
+2. Edit your existing Android SDK setup from [RUM Android Monitoring][3].
+3. Add tracking for web views with the following example:
+
+   ```
+   val configuration = Configuration.Builder(
```

Comment:
indent should be consistent with lines below (`val credentials` and `Datadog.initialize`)